packages/sdks/core-js-sdk/src/sdk/types.ts

```diff
@@ -335,6 +335,8 @@ export type Options = {
   oidcPrompt?: string;
   oidcErrorRedirectUri?: string;
   nativeOptions?: NativeOptions;
+  thirdPartyAppStateId?: string;
+  scopes?: string;
```

Comment:
Shouldn’t we give it a more meaningful name? 
It might be difficult to understand the purpose of this parameter later on

Reply:
Yea, I missed that.. adding now.

Reply:
My concern is that in a few months from now, when we will want to refactor the web-component, it will be hard to tell which params are related to which feature

Reply:
At least let's add a comment that this param is related to the third party apps

Reply:
Renamed and added comment as per your request (I understood its important for you so..:) ) 
please review.